src/custom/state/gas/hooks.ts

```diff
@@ -0,0 +1,34 @@
+import { ChainId } from '@uniswap/sdk'
+import { useCallback } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { GasState } from './reducer'
+import { updateGasPrices, UpdateGasPrices } from './actions'
+import { AppDispatch } from 'state'
+import { AppState } from '..'
+import { DEFAULT_NETWORK_FOR_LISTS } from 'constants/lists'
+import { GAS_FEE_ENDPOINTS } from 'constants/index'
+
+export interface GasFeeEndpointResponse {
+  lastUpdate: string
+  lowest: string
+  safeLow: string
+  standard: string
+  fast: string
+  fastest: string
+}
+
+export async function getGasPrices(chainId: ChainId = DEFAULT_NETWORK_FOR_LISTS): Promise<GasFeeEndpointResponse> {
```

Comment:
nit: in the operator, the API endpoint impl we did it in the utils. Im fine leaving it here anyways since this hook is small

Reply:
yeah i kept it here because of it's so small but agree otherwise..